src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -85,6 +85,7 @@ def make_artifacts(
     "--seed",
     type=str,
     default="",
+    show_default=True,
```

Comment:
What does show_default do?

Reply:
Just shows what the default value is when you `--help`